Standards/scs-0004-w1-achieving-certification-implementation.md

```diff
@@ -0,0 +1,133 @@
+---
+title: "Implementtaion hints for achieveing SCS-compatible certification"
+type: Supplement
+track: Global
+status: Draft
+supplements:
+  - scs-0004-v1-achieving-certification.md
+---
+
+## Process overview
+
+The *SCS-compatible* Certification for Operators is a technical certification:
+The Operator needs to fulfill technical requirements, such as providing certain
+APIs and guaranteeing certain platform behavior in order to be certifiable.
+
+These requirements are meant to provide guarantees to their customers, allowing
+them to rely on certain features to be available and on certain system behavior
+that lets their applications run in a reliable way.
+
+The SCS certification process typically consists of a few simple steps:
+
+1. Running the SCS compliance test suite and adjusting the infrastructure until it passes.
+2. Any additional declarations (for non-testable aspects) are written and passed to the SCS certification body.
+3. The operator must be a member ("shaper" or "advisor" level) of the Forum SCS-Standards in the
+   OSB Alliance (a non-profit) and pay the respective membership fees. Alternatively fees can
+   be paid without becoming a member.
+4. The cloud can be listed on the SCS pages as *SCS-compatible* with a compatibility status that is
+   updated on a daily basis. SCS then tests the infrastructure on a daily basis.
+
+The precise rules that govern how certificates are issued or withdrawn are defined in the
+[SCS standard 0004](scs-0004-v1-achieving-certification.md).
+
+## Self-testing and technical adjustments
+
+In order for a cloud service offering to obtain a certificate, it has to
+conform to all standards of the respective scope, which will be tested at
+regular intervals, and the results of these tests will be made available
+publicly.
+
+The best approach to get your cloud into compliance is by installing the
+test suite locally. Have a look at the
+[blog article](https://scs.community/blog/2024/10/14/cert-adapt-example/).
+
+A description how *SCS-compatible IaaS* compliance can be achieved on environments that use different
+OpenStack implementations is written up in a blog article
+[Cost of making an OpenStack Cluster SCS compliant](https://scs.community/2024/05/13/cost-of-making-an-openstack-cluster-scs-compliant/).
+
+## Declarations
+
+For the SCS-compatible IaaS v5 standard, the providers must — if they implement availability zones
+at all (which is optional) — guarantee certain levels of independence for these. This can not
+be fully tested by an automated test. The process thus envisions that providers must create some
+documentation on the physical infrastructure and how it maps to availability zones and declare that
+this documentation reflects the truth. SCS will review the docs and judge whether they meet the
+criteria. In case of doubt, audits can be performed.
```

Comment:
This should be dropped, because it is redundant and it ages too quickly. Suggestion:

- make the respective test output a note that documentation is required (mind you that it will show up as MISSING anyway, because the automated tests won't produce anything for the test cases in question)

On second thought, this is probably what is going to happen anyway. This paragraph is just a bit ahead of its time. Still, it should be replaced by a general remark that some test cases will turn up missing because documentation must be handed in.